components/bl-leafletMap-component/component.json

```diff
@@ -0,0 +1,320 @@
+{
+  "id": "c_fd8c99a633b33a236df39c4b2a76f8c2",
+  "name": "Leaflet map",
+  "description": "The component allows you to work with the map.",
+  "showInToolbox": true,
+  "faIcon": "map",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "checkbox",
+      "name": "mapVisibility",
+      "label": "Map Visibility",
+      "defaultValue": true,
+      "handlerId": "mapVisibilityHandler",
+      "handlerLabel": "Map Visibility Logic",
+      "handlerDescription": "Used to control component visibility.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "center",
+      "label": "Center",
+      "defaultValue": "40.74, -73.96",
```

Comment:
Could we use here some popular place? Statue of liberty, eiffel tower or something else?